.scribe/endpoints/02.yaml

```diff
@@ -56,7 +56,7 @@ endpoints:
       custom: []
     headers:
       Accept: application/json
-      Referer: learnhub.mk
+      Referer: myapi.local
```

Comment:
We don't need this change. Make sure to use learnhub.mk for the stateful domains and the session domain environment variables locally when generating the docs